cmd/veneur-emit/main.go

```diff
@@ -56,9 +64,23 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Error!")
 	}
+	_ = conn
+
+	var nconn net.Conn
+	nconn, err = net.Dial("udp", addr)
+	if err != nil {
+		logrus.WithError(err).Fatal("Error!")
+	}
 
-	tags := tags(*tag)
-	err = sendMetrics(conn, passedFlags, *name, tags)
+	// tags := tags(*tag)
+
+	var span *ssf.SSFSpan
```

Comment:
That this is now *the* way with veneur-emit, we're in a chicken and egg problem. You asked if we should use a flag for this? I think we probably should. If we don't, we have to fully deploy the "new" SSF-capable veneur before this version of veneur-emit can go out. If we try and wait, we run the risk that finding a bug in veneur-emit can't be patched off master. :)